concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md

```diff
@@ -1,27 +1,27 @@
-You can use SVG icons instead of font icons. The following code demonstrates how to use them in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used for widgets with the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') option. 
+In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme repository</a> on GitHub.
 
-An SVG icon can be specified as follows:
+The following code demonstrates how to use SVG icons in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used with any other UI component that has the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') property. 
 
-1. **By the URL**:
+1. **Use the icon's URL**:
 
         <!-- tab: JavaScript -->
         new DevExpress.ui.dxButton(targetElement, {
-            icon: "https://js.devexpress.com/Content/data/loadingIcons/rolling.svg"
+            icon: "https://path/to/the/icon.svg"
         });
 
-1. **Inserted inline**:
+1. **Insert SVG content inline**:
 
         <!-- tab: JavaScript -->
         new DevExpress.ui.dxButton(targetElement, {
             icon: "<svg>SVG_CONTENT</svg>"
         });
 
-1. **Imported**:
+1. **Import the icon**:
 
         <!-- tab: JavaScript -->
         import * as myIcon from "./assets/icon.svg";
         new DevExpress.ui.dxButton(targetElement, {
             icon: myIcon
         });
 
-[important]The SVG format allows you to run executable code that might be malicious. It is strongly recommended that you use SVG icons only from trusted sources.
+[important]The SVG format allows running executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
```

Comment:
1) `The SVG format allows running executable code that might be malicious.`

The original sentence specified _who_ ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.
Can we go back to the original sentence ("allows you to run executable code")?

2) `We strongly recommend that you use SVG icons only from trusted sources.`

This sentence is fine.

Reply:
> The original sentence specified who ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.

That was the motivation behind this change. It is not a common practice to put any executable code in SVG. So, if an SVG icon contains code, is is most likely malicious, and the reader never wants to run it.

Reply:
Thanks for the clarification. I understand now. Can we say the following?

`The SVG format could allow malicious code to run. We strongly recommend that you use SVG icons only from trusted sources.`